### PR TITLE
[k8s.io/client-go/discovery/helper]: fixup typo

### PR DESCRIPTION
**Pull request by ideapark**
Signed-off-by: Zhou Peng <p@ctriple.cn>

**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespace from that line:
>
> /kind api-change
> /kind bug

/kind cleanup

> /kind deprecation
> /kind design
> /kind documentation
> /kind failing-test
> /kind feature
> /kind flake

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note none

```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs

```

### CODE REVIEW

**Review by hasheddan — APPROVED** [review]
/lgtm
/retest

File: staging/src/k8s.io/client-go/discovery/helper.go [hunk 1]
```diff
@@ -114,7 +114,7 @@ func (fn ResourcePredicateFunc) Match(groupVersion string, r *metav1.APIResource
 	return fn(groupVersion, r)
 }
 
-// SupportsAllVerbs is a predicate matching a resource iff all given verbs are supported.
+// SupportsAllVerbs is a predicate matching a resource if all given verbs are supported.
```
Comment (caesarxuchao) [inline staging/src/k8s.io/client-go/discovery/helper.go:117]: The meaning of `iff` here is `if and only if`. It seems that`iff` is not a well-known acronym, let's expand it to the full phrase here.

**Pull request closed without merging**